### ISSUE TITLE
Accesibility support for the disabled

### ISSUE THREAD

**Issue by 254v0lkan** [ampproject/amphtml#21536]
##Are the current AMP tags friendly to the disabled?

The blind, color-blind and other people need access to the content in this pages, is it supported?

**Comment by aghassemi** [comment]
Hi @254v0lkan 

Yes, we [ensure](https://github.com/ampproject/amphtml/issues?q=is%3Aissue+label%3A%22Related+to%3A+Accessibility%22+is%3Aclosed) built-in AMP components are accessible and page authors have all the normal HTML tools such as `aria-*` rules to make their content accessible. The degree of accessibility of an AMP page also depends on how much the page author has built a11y into their code. 

to @nainar who is working on writing documentation around this topic.

**Comment by nainar** [comment]
Hi @254v0lkan like @aghassemi mentioned like any other framework while AMP does allow you to create accessible experiences we can't enforce that all websites using AMP are meeting WCAG guidance. For example, while <amp-img> does allow you to specify an alternate text, we don't enforce that all images have alternate text; even if we did, we couldn’t enforce that the string provided was actually a description of the image.

**Comment by 254v0lkan** [comment]
Hi @aghassemi and @nainar  , thanks for the feedback

How can I contribute?

**Comment by nainar** [comment]
Hi @254v0lkan this [link](https://github.com/ampproject/amphtml/blob/master/CONTRIBUTING.md#get-started-with-open-source) should help you get started. Please use our [Slack channel](https://github.com/ampproject/amphtml/blob/master/CONTRIBUTING.md#discussion-channels) for questions. I will close this issue since we use GitHub to discuss technical issues with our codebase. Thank you for your interest!
